ethereum/contracts/IncentivizedOutboundChannel.sol

```diff
@@ -7,10 +7,22 @@ import "./OutboundChannel.sol";
 // IncentivizedOutboundChannel is a channel that sends ordered messages with an increasing nonce. It will have incentivization too.
 contract IncentivizedOutboundChannel is OutboundChannel {
 
-    constructor() {
+    uint256 public gasFee;
```

Comment:
```suggestion
    uint256 public fee;
```
This terminology may be a bit misleading. The fee here isn't gas paid to Ethereum network miners for quicker relay, but rather is paid to Relayers. I think `fee` or `relayFee` may be clearer.

Reply:
yeh relayFee is good